scripts/install.py

```diff
@@ -793,7 +807,11 @@ def do_backup():
         if is_user(OPTS.user):
             OPTS.sudo_user = ''
         else:
-            exit_with_error(1, "Installation expects you to be root for full installation or user for upgrade. Make sure --user and --group are set correctly.")
+            exit_with_error(
+                1,
+                ("Installation expects you to be root for full installation or user for upgrade. "
+                 "Make sure --user and --group are set correctly.")
```

Comment:
I thought that the parentheses really are not needed here?
```suggestion
                "Installation expects you to be root for full installation or user for upgrade. "
                 "Make sure --user and --group are set correctly."
```